### DISCUSSION TITLE
Azure AD authentication is possible in akhq ?

### DISCUSSION THREAD

**Discussion by ankurjains** [tchiotludo/akhq#868] [General]
I have azure ad user and roles and I am looking to provide akhq running in aks AD USERS (kind of sso also) is that currently a way to do in akhq ?

**Comment by tchiotludo** [comment]
I don't have any azure account, but as I see, you should use [oidc](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-oidc-sso)

**Reply by ankurjains** [reply]
inputs on how it can be configured in akhq for azure provider

**Reply by tchiotludo** [reply]
no clue but a quick google search led me to this : 
You will have to mix this [one to configure your ad](https://docs.microsoft.com/en-us/powerapps/maker/portals/configure/configure-openid-settings) and this [one to configure micronaut on akhq](https://micronaut-projects.github.io/micronaut-security/latest/guide/#openid)

**Reply by ankurjains** [reply]
I am not java dev it seems coding also needs to be added for this to make work

**Reply by tchiotludo** [reply]
Can you elaborate a bit more please ? 
Your sentence is equal to "it don't work" and "it don't help"

**Reply by ankurjains** [reply]
I am assuming only editing application.yaml is enough to configure odic with azure ad or do I need to add java code to configure it as well from one to configure micronaut on akhq ?

**Reply by tchiotludo** [reply]
editing yaml is suffisant

**Reply by ankurjains** [reply]
nice
